### PR TITLE
Fixing pod spec to work with recent changes 

### PR DESCRIPTION
**Pull request by wmathurin**
* fmdb sources coming from submodule
* fts support
NB: the most recent sqlcipher available through cocoapod is sqlcipher 3.1, but the sdk ships with sqlcipher 3.3

### CODE REVIEW

File: SalesforceMobileSDK-iOS.podspec [hunk 1]
```diff
@@ -74,7 +74,8 @@ Pod::Spec.new do |s|
       sdkcore.dependency 'SalesforceMobileSDK-iOS/SalesforceOAuth'
       sdkcore.dependency 'OpenSSL-Universal', '1.0.1.k'
       sdkcore.dependency 'SQLCipher', '~> 3.1'
-      sdkcore.source_files = 'libs/SalesforceSDKCore/SalesforceSDKCore/Classes/**/*.{h,m}', 'libs/SalesforceSDKCore/SalesforceSDKCore/SalesforceSDKCore.h'
+      sdkcore.dependency 'SQLCipher/fts', '~> 3.1'
```
Comment (wmathurin) [inline SalesforceMobileSDK-iOS.podspec]: They have a subspec for fts

File: SalesforceMobileSDK-iOS.podspec [hunk 2]
```diff
@@ -74,7 +74,8 @@ Pod::Spec.new do |s|
...
+      sdkcore.source_files = 'libs/SalesforceSDKCore/SalesforceSDKCore/Classes/**/*.{h,m}', 'libs/SalesforceSDKCore/SalesforceSDKCore/SalesforceSDKCore.h', 'external/fmdb/src/fmdb/*.{h,m}'
```
Comment (wmathurin) [inline SalesforceMobileSDK-iOS.podspec]: FMDB sources now come from submodule

File: SalesforceMobileSDK-iOS.podspec [hunk 3]
```diff
@@ -74,7 +74,8 @@ Pod::Spec.new do |s|
       sdkcore.dependency 'SalesforceMobileSDK-iOS/SalesforceOAuth'
       sdkcore.dependency 'OpenSSL-Universal', '1.0.1.k'
       sdkcore.dependency 'SQLCipher', '~> 3.1'
```
Comment (bhariharan) [inline SalesforceMobileSDK-iOS.podspec]: Should the ```SQLCipher``` dependency in the line above remain, or should it go, now that we have the ```SQLCipher/fts``` dependency?

**Comment by bhariharan** [comment]
Looks good to me. :+1:

**Comment by khawkins** [comment]
Seems okay to me. :+1:

File: SalesforceMobileSDK-iOS.podspec (hunk 3 above)
Comment (wmathurin) [inline SalesforceMobileSDK-iOS.podspec]: You are probably right, do we care one way or the other?

Comment (bhariharan) [inline SalesforceMobileSDK-iOS.podspec]: If they don't conflict, then I don't care either way :-)

**Pull request merged**